packages/graphiql/src/css/app.css

```diff
@@ -401,17 +383,12 @@
 }
 
 .graphiql-container .CodeMirror-scroll {
-  overflow-scrolling: touch;
+  -webkit-overflow-scrolling: touch;
```

Comment:
`overflow-scrolling` doesn't exist, so changed it to `-webkit-overflow-scrolling`